static/styles/popovers.css

```diff
@@ -833,3 +833,104 @@ ul {
         column-count: 1;
     }
 }
+
+.popover-menu {
```

Comment:
This class name seems overly generic -- is this intended to be styles for a broad class of menus, or just this one? 

I worry a bit that this sounds a lot like it has to do with the `.popover-content` type CSS classes we use for other popovers... but maybe that's fine if we think we're going to migrate other popovers in this direction visually. (Deserves a comment, though).

Reply:
I agree that name is confusing.
I changed ".popover-menu__list" to ".popover-list."
A BEM layer was removed without changing the design.